src/templates/viewer-spec.yaml.j2

```diff
@@ -0,0 +1,38 @@
+# Source: manifests/apps/volumes-web-app/upstream/base/viewer-spec.yaml
+# Note: the volumes-web-app allows expanding strings using ${VAR_NAME}
+# You may use any environment variable. This lets us e.g. specify images that can be modified using kustomize's image transformer.
+# Additionally, 'PVC_NAME', 'NAME' and 'NAMESPACE' are defined
```

Comment:
I am assuming that the volumes web app will be rendering $NAME, $NAMESPACE at runtime(?), is any of this something we should render with information from the charm?

Reply:
These are passed to `viewer-spec.yaml` during runtime [here](https://github.com/kubeflow/kubeflow/blob/f4f8bf14fb00d013d351d2d25643f59f1338ab77/components/crud-web-apps/volumes/backend/apps/common/viewer.py#L34-L37) and defined and passed to it [here](https://github.com/kubeflow/kubeflow/blob/f4f8bf14fb00d013d351d2d25643f59f1338ab77/components/crud-web-apps/volumes/backend/apps/default/routes/post.py#L34-L35) so no need to define those.

Reply:
Interesting.